tests/integration/ioredis.Dockerfile

```diff
@@ -29,6 +30,6 @@ ADD .run_ioredis_valid_test.sh run_tests.sh
 
 # this would enable running monitor commands successfully
 # we may need to remove this incase we have other fix
-RUN ./patch_ioredis.sh test/functional/monitor.ts
+#RUN ./patch_ioredis.sh test/functional/monitor.ts
```

Comment:
and this ?

Reply:
we don't need this any more - because the latest version in ioredis solved it